en/ha.rst

```diff
@@ -3964,6 +3964,8 @@ The following are optional items:
 
 *   **scp_option** : Specifies the **scp** option which enables backup of source node in which replication is rebuilt to copy into the target node. The default option is **-l 131072**, which does not impose an overload on network (limits the transfer rate to 16 MB).
 
+*   **ssh_port**  : Specifies the **port** number of the ssh and scp used in the script. This option also applies to **expect** run from the script. The default port number is **22**.
```

Comment:
Why do sentences in the same paragraph start with a capital letter?

Reply:
Capitalization After Colons
1. When a colon introduces a list of of things, do not capitalize the first word after the colon unless it is a proper noun.
2. When a colon introduces a phrase or an incomplete sentence that is meant to add information to the sentence before it, do not capitalize the first word after the colon unless it is a proper noun.

Therefore, there 'Specifies' should change to 'specifies', the lower case

Reply:
OK, that capitalization was just for consistency with descriptions above. 
I'll make another issue to fix capitalization misuse